### PR TITLE
Add histogram

### PR DESCRIPTION
**Pull request by gsralex**
## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [ ] GitHub Actions works fine on your own branch.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsKey.java [hunk 1]
```diff
@@ -47,6 +47,9 @@ public enum MetricsKey {
     METRIC_RT_P99("dubbo.%s.rt.seconds.p99", "Response Time P99"),
     METRIC_RT_P95("dubbo.%s.rt.seconds.p95", "Response Time P95"),
 
+    PROVIDER_METRIC_RT_HISTOGRAM("dubbo.provider.rt.seconds.histogram", "Response Time Histogram"),
```
Comment (songxiaosheng) [inline dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsKey.java:50]: please change provider to %s

**Review by songxiaosheng — CHANGES_REQUESTED** [review]
please add test

File: dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsKey.java (hunk 1 above)
Comment (gsralex) [reply dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsKey.java]: modified

File: dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsKey.java [hunk 2]
```diff
@@ -47,6 +47,9 @@ public enum MetricsKey {
     METRIC_RT_P99("dubbo.%s.rt.seconds.p99", "Response Time P99"),
     METRIC_RT_P95("dubbo.%s.rt.seconds.p95", "Response Time P95"),
 
+    PROVIDER_METRIC_RT_HISTOGRAM("dubbo.%s.rt.seconds.histogram", "Response Time Histogram"),
```
Comment (songxiaosheng) [inline dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsKey.java:50]: please change the name  of PROVIDEER_METRIC_RT_HISTORYGRAM to   
METRIC_RT_HISTORYGRAM

seconds may be error , need change to  milliseconds

**Review by songxiaosheng — CHANGES_REQUESTED** [review]

**Comment by AlbumenJ** [comment]
image

**Review by AlbumenJ — COMMENTED** [review]
`dubbo-common/.git.exec.error` remove this file

**Comment by AlbumenJ** [comment]
image

**Comment by gsralex** [comment]
> image

modified

File: dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsKey.java (hunk 2 above)
Comment (gsralex) [reply dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsKey.java]: modified

**Comment by songxiaosheng** [comment]
@AlbumenJ  PTAL help check ci

**Review by AlbumenJ — APPROVED** [review]

**Comment by AlbumenJ** [comment]
Workflow runs failed.

image

**Comment by gsralex** [comment]
> Workflow runs failed.
> 
> image

fixed

**Comment by AlbumenJ** [comment]
image

**Comment by gsralex** [comment]
example:


dubbo.metrics.histogram.enabled=true
image


dubbo.metrics.histogram.enabled=false
image

**Review by songxiaosheng — APPROVED** [review]
LGTM

**Pull request merged**